backend/api/completions.ts

```diff
@@ -60,3 +66,96 @@ export function completions({ knex }: ApiContext) {
     req.on("close", stream.end.bind(stream))
   }
 }
+
+export function completionInstructions({ knex }: ApiContext) {
+  return async function (req: any, res: any) {
+    const id = req.params.id
+
+    const course = (
+      await knex.select<any, Course[]>("id").from("course").where("slug", id)
+    )[0]
+
+    const instructions = (
+      await knex
+        .select<any, CourseTranslation[]>("instructions")
+        .from("course_translation")
+        .where("course_id", course.id)
+    )[0]?.instructions
```

Comment:
Shouldn't this pick instructions by some language instead of the first one it encounters? `course_translation` generally has only English or Finnish available (in the editor anyway), so that might not be so optimal place to store it.

Reply:
Good point. BAI is in English (for now) and ECTS is for English, Finnish, and Swedish students. I've added a language param to the course instructions endpoint to select between the available course languages.